src/facade/reply_builder.h

```diff
@@ -24,8 +24,8 @@ enum class ReplyMode {
 class SinkReplyBuilder {
  public:
   struct ResponseValue {
-    std::string key;
-    std::string value;
+    std::string_view key;
```

Comment:
I wonder if it's ok with CapturingReplyBuilder, @dranikpg  ?

Reply:
No, it's not, it requires always owning types.

The options are
* Implementing a proxy object and converting to when storing the response
* Implementing SendMget as SendArray() + SendBulkString and letting it be captured automatically

Btw, there are lots of missing optimizations in the reply captures by themselves, but I don't think MGET is a frequent single-shard hit

Reply:
I chose a third way 🤷🏼